benchmark/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿using BenchmarkDotNet.Running;
+
+namespace Benchmark
+{
+    class Program
+    {
+        static void Main(string[] args) => BenchmarkRunner.Run<ParseBenchmark>();

```

Comment:
It seems we can remove `string[] args`.
Also, please add modifiers to the class and Main method